shared/src/ledger/eth_bridge/bridge_pool_vp.rs

```diff
@@ -77,6 +80,44 @@ where
             Some(SignedAmount::Positive(after - before))
         }
     }
+
+    /// Check that the correct amount of Nam was sent
+    /// from the correct account into escrow
+    fn check_nam_escrowed(
+        &self,
+        payer_account: &Address,
+        escrow_account: &Address,
+        expected_debit: Amount,
+        expected_credit: Amount,
+    ) -> bool {
+        // check that the correct amount was deducted from the fee payer
+        if let Some(SignedAmount::Negative(amount)) =
+            self.account_balance_delta(payer_account)
+        {
+            if amount != expected_debit {
+                return false;
+            }
+        } else {
+            tracing::debug!("The account {} was not debited.", payer_account);
+            return false;
+        }
+        // check that the correct amount was credited to escrow
+        if let Some(SignedAmount::Positive(amount)) =
+            self.account_balance_delta(escrow_account)
+        {
+            if amount != expected_credit {
+                return false;
+            }
+        } else {
+            tracing::debug!(
+                "The Ethereum bridge pool's escrow was not credited from \
+                 account {}.",
+                payer_account
+            );
+            return false;
+        }
+        true
```

Comment:
If any `self.account_balance_delta` return `None`s here, we would reach this `true`

Reply:
true, easy fix